docs/cases/cases-overview.asciidoc

```diff
@@ -3,172 +3,15 @@
 
 = Cases
 
-Cases are used to open and track security issues directly in the {es-sec-app}.
-All cases list the original reporter and all users who contribute to a case
-(`participants`). Comments support Markdown syntax and allow linking to saved <<timelines-ui,Timelines>>. Additionally, you can send cases to these
-external systems from within {es-sec}:
+Cases are used to open and track security issues directly in the {es-sec-app}. All cases list the original reporter and all users who contribute to a case (`participants`). Comments support Markdown syntax and allow linking to saved <<timelines-ui,Timelines>>. You can also send cases to these external systems by <<cases-ui-integrations, configuring external connectors>>:
```

Comment:
Unclear why "participants" is code format, or why it's mentioned here at all. This is just a general overview of what cases are, and for that it seems like "users who contribute to a case" is clear enough. If necessary, the more focused sub-topic on using the Cases UI can get into detail about the "Participants" section of the UI.
```suggestion
Cases are used to open and track security issues directly in the {es-sec-app}. All cases list the original reporter and all users who contribute to a case. Comments support Markdown syntax and allow linking to saved <<timelines-ui,Timelines>>. You can also send cases to these external systems by <<cases-ui-integrations, configuring external connectors>>:
```

Reply:
@joepeeples I don't have any issues removing "participants". The detail came from the original case docs and I've been wanting to update it (the whole intro paragraph, actually) to something more generic like:

Collect and share information about security issues by opening a case in Elastic Security. Cases allow you to track key investigation details, case metrics, and more. The Elastic Security UI provides several ways to create and manage cases. Alternatively, you can use the [Cases API](https://www.elastic.co/guide/en/security/master/cases-api-overview.html) to perform the same tasks.

WDTY?

Reply:
New intro sounds good! It might be nice to also mention alerts, since I imagine that's a pretty common workflow for the SOC analyst role: triage alerts, then create cases based on the alerts or attach the alerts to existing cases.